amd_openvx_extensions/amd_rpp/source/Brightness.cpp

```diff
@@ -152,209 +152,85 @@ static vx_status VX_CALLBACK validateBrightness(vx_node node, const vx_reference
     return status;
 }
 
-static vx_status VX_CALLBACK processBrightness(vx_node node, const vx_reference *parameters, vx_uint32 num)
-{
+static vx_status VX_CALLBACK processBrightness(vx_node node, const vx_reference *parameters, vx_uint32 num) {
```

Comment:
Parameter num is not used anywhere else in this function
Please check if this can be removed

Reply:
This is by default present in all process calls of AMD OpenVX extensions so we cannot remove it here